### PR TITLE
[Refactor] rename configs and update readme accordingly

### PR DESCRIPTION
**Pull request by zengyh1900**

### CODE REVIEW

File: configs/edvr/edvrm_reds_600k-8xb8.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
-_base_ = '../_base_/models/base_edvr_config.py'
```
Comment (LeoXing1996) [inline configs/edvr/edvrm_reds_600k-8xb8.py:1]: `x4`

File: configs/gca/baseline_dimaug-r34_200k-4xb10_comp1k.py [hunk 2]
```diff
@@ -1,4 +1,8 @@
-_base_ = ['./baseline_r34_4x10_200k_comp1k.py']
+_base_ = ['./baseline_r34_200k-4xb10_comp1k.py']
+
+experiment_name = 'baseline_dimaug-r34_200k-4xb10_comp1k'
```
Comment (LeoXing1996) [inline configs/gca/baseline_dimaug-r34_200k-4xb10_comp1k.py:3]: `dimaug` is related to augmentation. Maybe place this in `training setting`?

File: configs/gca/gca_dimaug-r34_200k-4xb10_comp1k.py [hunk 3]
```diff
@@ -1,4 +1,8 @@
-_base_ = ['./gca_r34_4x10_200k_comp1k.py']
+_base_ = ['./gca_r34_200k-4xb10_comp1k.py']
+
+experiment_name = 'gca_dimaug-r34_200k-4xb10_comp1k'
```
Comment (LeoXing1996) [inline configs/gca/gca_dimaug-r34_200k-4xb10_comp1k.py:3]: `dimaug` is related to augmentation. Maybe place this in `training setting`?

File: configs/glean/glean_2xb8_ffhq-x16.py [hunk 4]
```diff
@@ -1,7 +1,8 @@
 _base_ = '../_base_/models/base_glean.py'
 
-experiment_name = 'glean_ffhq_16x'
+experiment_name = 'glean_2xb8_ffhq-x16'
```
Comment (LeoXing1996) [inline configs/glean/glean_2xb8_ffhq-x16.py:3]: `x16`

File: configs/glean/glean_2xb8_cat-x8.py [hunk 5]
```diff
@@ -1,7 +1,8 @@
 _base_ = '../_base_/models/base_glean.py'
 
-experiment_name = 'glean_cat_8x'
+experiment_name = 'glean_2xb8_cat-x8'
```
Comment (LeoXing1996) [inline configs/glean/glean_2xb8_cat-x8.py:3]: `x8`

File: configs/glean/glean_in128out1024_300k-4xb2_ffhq-celebahq.py [hunk 6]
```diff
@@ -1,7 +1,8 @@
 _base_ = '../_base_/models/base_glean.py'
 
-experiment_name = 'glean_in128out1024_4x2_300k_ffhq_celebahq'
+experiment_name = 'glean_in128out1024_300k-4xb2_ffhq-celebahq'
```
Comment (LeoXing1996) [inline configs/glean/glean_in128out1024_300k-4xb2_ffhq-celebahq.py:3]: `ffhq-celeba-HxW`

File: configs/edvr/edvrm_reds_600k-8xb8.py (hunk 1 above)
Comment (zengyh1900) [reply configs/edvr/edvrm_reds_600k-8xb8.py:1]: ok

File: configs/gca/baseline_dimaug-r34_200k-4xb10_comp1k.py (hunk 2 above)
Comment (zengyh1900) [reply configs/gca/baseline_dimaug-r34_200k-4xb10_comp1k.py:3]: ok

File: configs/gca/gca_dimaug-r34_200k-4xb10_comp1k.py (hunk 3 above)
Comment (zengyh1900) [reply configs/gca/gca_dimaug-r34_200k-4xb10_comp1k.py:3]: ok

File: configs/glean/glean_2xb8_ffhq-x16.py (hunk 4 above)
Comment (zengyh1900) [reply configs/glean/glean_2xb8_ffhq-x16.py:3]: x16 has been placed with dataset

File: configs/glean/glean_2xb8_cat-x8.py (hunk 5 above)
Comment (zengyh1900) [reply configs/glean/glean_2xb8_cat-x8.py:3]: x8 has been placed with dataset

File: configs/glean/glean_in128out1024_300k-4xb2_ffhq-celebahq.py (hunk 6 above)
Comment (zengyh1900) [reply configs/glean/glean_in128out1024_300k-4xb2_ffhq-celebahq.py:3]: in128out1024 refer to the structure of networks, and we only list dataset here, resolution is not necessary

File: configs/ttsr/ttsr-gan_c64b16_500k-1xb9_CUFED-x4.py [hunk 7]
```diff
@@ -1,8 +1,8 @@
-_base_ = './ttsr-rec_x4_c64b16_g1_200k_CUFED.py'
+_base_ = './ttsr-rec_c64b16_200k-1xb9_CUFED-x4.py'
 
-experiment_name = 'ttsr-gan_x4_c64b16_g1_500k_CUFED'
+experiment_name = 'ttsr-gan_c64b16_500k-1xb9_CUFED-x4'
```
Comment (LeoXing1996) [inline configs/ttsr/ttsr-gan_c64b16_500k-1xb9_CUFED-x4.py:3]: `x4`

File: configs/tof/tof_spynet-sintel-wobn-clean_1xb1_vimeo90k-triplet.py [hunk 8]
```diff
@@ -1,7 +1,8 @@
-_base_ = '../_base_/models/base_tof_vfi_nobn_1xb1_vimeo90k_triplet.py'
+_base_ = '../_base_/models/base_tof.py'
 
-experiment_name = 'tof_vfi_spynet_clean_nobn_1xb1_vimeo90k'
```
Comment (LeoXing1996) [inline configs/tof/tof_spynet-sintel-wobn-clean_1xb1_vimeo90k-triplet.py:3]: What does `vfi` mean?

Comment (zengyh1900) [reply configs/tof/tof_spynet-sintel-wobn-clean_1xb1_vimeo90k-triplet.py:3]: task name: video frame interpolation. 
For now, we remove task name in config filename

**Pull request merged**